PKGBUILD

```diff
@@ -0,0 +1,56 @@
+# Maintainer: Jonas Renk <jonas.renk@stud.tu-darmstadt.de>
+pkgname=systemverilog
+pkgver=20210116
+pkgrel=1
+epoch=
+pkgdesc="Synthese und Simulation"
+arch=('any')
+url="https://moodle.informatik.tu-darmstadt.de/mod/folder/view.php?id=31324"
```

Comment:
As you have already remarked in your coment regarding this pull request, using a URI that is not publicly accessible does not seem to do the trick. A possible alternative would be referencing the website that provides general information about the course "Digitaltechnik", which can be found here: https://www.encrypto.cs.tu-darmstadt.de/teaching_theses/digitaltechnik_ws2021/index.en.jsp

Reply:
Your URL should to the trick.